### ISSUE TITLE
[bug] After opening a file in a tab, closing that tab also closes the DiffView tab

### ISSUE THREAD

**Issue by joeyschoblaska** [sindrets/diffview.nvim#223]
If I'm viewing a diff, open a file in a new tab (`<C-w>gf`), and then close the tab for that file, DiffView also closes. Maybe this is expected behavior, but it was surprising to me given some other behaviors, eg:

1. Open a file in a tab
2. Close the DiffView tab
3. The file's tab remains open

And:

1. Open a file "normally" (`gf`)
2. Close that file's split
3. DiffView remains open

https://user-images.githubusercontent.com/29129/191997460-4dd755ec-c262-401b-aa5c-1acc8c7cfb62.mp4

Thanks for the excellent plugin!

**Comment by sindrets** [comment]
I can't reproduce this. There's likely something in your config that's interfering (perhaps an auto command or a plugin). Please provide a minimal config that makes this reproducible. [Here's a template](https://gist.github.com/sindrets/3e671e79ce4566667f9ec176786dfadd).

**Comment by joeyschoblaska** [comment]
Hmm, you're right. I can't reproduce with the minimal config either. I'll close this issue and comment if I figure out what the problem was. Thanks!

**Comment by joeyschoblaska** [comment]
I removed all of my plugins and verified that Diffview was now working as expected. I then progressively added my plugins back to see which one was the culprit. Eventually I got them all added back and... Diffview was still working normally. So maybe I had something strange in my compiled plugin cache or something, but my config is unchanged from when I observed the bug except now Diffview works fine. 🤷
